client/src/helpers/APICalls/uploadImage.ts

```diff
@@ -0,0 +1,17 @@
+import { ImageUploadData } from '../../interface/File';
+import { IUploadOptions } from '../../interface/File';
+
+const uploadImage = async (form: FormData): Promise<ImageUploadData> => {
+  const fetchOptions: IUploadOptions = {
+    method: 'POST',
+    body: form,
+  };
+
+  return fetch(`http://localhost:3001/files/upload`, fetchOptions)
```

Comment:
Do we need to refer the entire url localhost:3001? since proxy is set up ideally we just want /files/upload

Reply:
Oh yes, sure!